src/guacd-docker/krb5.conf

```diff
@@ -0,0 +1,16 @@
+[libdefaults]
+    default_realm = PROCYON.COM
+    dns_lookup_kdc = true
+    dns_lookup_realm = true
+
+[realms]
+    PROCYON.COM = {
+        kdc = cez-private.procyon.com
+        admin_server = cez-private.procyon.com
+        master_kdc = cez-private.procyon.com
+        default_domain = procyon.com
+    }
+
+[domain_realm]
+    .procyon.com = PROCYON.COM
+    procyon.com = PROCYON.COM
```

Comment:
I think this file should be removed from the commit - this looks like something internal to your configuration. I'm not sure that we should provide a template for this - maybe just document instructions for it.